Rakefile

```diff
@@ -23,6 +23,13 @@ end
 
 desc 'Setup and run all tests'
 task :setup_and_run_tests do
+  ENV['BUNDLE_GEMFILE'] = File.join(__dir__, 'gemfiles', "#{ENV['RAILS_VERSION']}.gemfile") if ENV['RAILS_VERSION']
```

Comment:
maybe it could default to the latest available rails Gemfile and display that it has done that automatically instead of exiting and forcing the user to set an ENV var ?

Reply:
@mathieujobin I've switched the Rakefile and GH actions to use `RAILS_VERSION` directly as otherwise `BUNDLE_GEMFILE` was defaulting to `wicked_pdf/Gemfile` which never actually works